sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultPrivateKey.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import java.security.PrivateKey;
+
+/**
+ * KeyVault fake private which work when key less
+ */
+public class KeyVaultPrivateKey implements PrivateKey {
+
+    /**
+     * Stores the serial version UID.
+     */
+    private static final long serialVersionUID = 30_10_00;
```

Comment:
Seems it's OK to delete serialVersionUID. 
I already deleted serialVersionUID in other classed in this PR: https://github.com/Azure/azure-sdk-for-java/pull/23200